integration/rest/src/main/java/it/gov/pagopa/rtd/transaction_filter/connector/HpanRestConnector.java

```diff
@@ -38,4 +39,7 @@ public interface HpanRestConnector {
 
     @GetMapping(value = "${rest-client.sender-ade-ack.download-file.url}")
     ResponseEntity<Resource> getSenderAdeAckFile(@RequestHeader("Ocp-Apim-Subscription-Key") String token, @PathVariable(name = "id") String fileName);
+
+    @PutMapping(value = "${rest-client.sender-ade-ack.received.url}")
+    ResponseEntity<Void> postAckReceived(@RequestHeader("Ocp-Apim-Subscription-Key") String token, @PathVariable(name = "id") String fileName);
```

Comment:
Shouldn't this method be called "putAckReceived"?

Reply:
Corrected, thank you.